backend/private-graph/graph/resolver.go

```diff
@@ -76,6 +76,8 @@ const ErrorGroupLookbackDays = 7
 const SessionActiveMetricName = "sessionActiveLength"
 const SessionProcessedMetricName = "sessionProcessed"
 
+const AuthorizationError = "403 - AuthorizationError"
```

Comment:
We can also add `401 - AuthenticationError` when someone isn't logged in:
https://github.com/highlight/highlight/blob/718d86dedddf1937c859959577e5b76cd005e06b/backend/private-graph/graph/schema.resolvers.go#L7217-L7220